fdbcli/GetAuditStatusCommand.actor.cpp

```diff
@@ -196,7 +198,7 @@ ACTOR Future<bool> getAuditStatusCommandActor(Database cx, std::vector<StringRef
 		}
 		const UID id = UID::fromString(tokens[3].toString());
 		AuditStorageState res = wait(getAuditState(cx, type, id));
-		printf("Audit result is:\n%s", res.toString().c_str());
+		fmt::println("Audit result is:\n{}", res.toString());
```

Comment:
Should we be using `fmt::print` here?

Reply:
I checked this, it needs to be `println`, otherwise last line gets buffered and not printed sometimes. See also the other Audit result prints later in the file also use println (had \n at the end in the past).